src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -264,7 +251,9 @@ NavHeader.propTypes = {
   placeholderText: PropTypes.string,
   setIsFirstTimeOpen: PropTypes.func,
   getSearchLayout: PropTypes.func,
-  hasSearch: PropTypes.bool
+  hasSearch: PropTypes.bool,
+  onFollowButtonPress: PropTypes.func,
+  followType: PropTypes.string
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `onFollowButtonPress` and `followType` to PropTypes is necessary for type checking. Good practice.



<!-- This is an auto-generated comment by OSS CodeRabbit -->